### PR TITLE
[thread-cert] add case 9.2.5 ActiveDataset using pktverify

### PR DESCRIPTION
**Pull request by FiveDimensions**

### CODE REVIEW

File: tests/scripts/thread-cert/Cert_9_2_05_ActiveDataset.py [hunk 1]
```diff
@@ -0,0 +1,368 @@
...
+        self.nodes[ROUTER].send_mgmt_active_set(
+            active_timestamp=100,
+            channel_mask=0x3fff800,
+            extended_panid='000db80000000001',
+            mesh_local='fd00:0db8::',
+            network_name='TEST_1',
+            master_key='00112233445566778899aabbccddeeff',
+            panid=0xface,
+            channel=11,
+            binary='0410d2aa9cd8dff7919122d77d37ec3c1b5f0c030e10e0',
```
[the remark this answers is not in the document]
Comment (simonlingoogle) [reply tests/scripts/thread-cert/Cert_9_2_05_ActiveDataset.py:105]: can we specify `pskc` argument?

File: tests/scripts/thread-cert/Cert_9_2_05_ActiveDataset.py [hunk 2]
```diff
@@ -0,0 +1,368 @@
...
+                              NM_NETWORK_NAME_TLV,
+                              NM_PAN_ID_TLV,
+                              NM_PSKC_TLV,
+                              NM_SECURITY_POLICY_TLV
+                             } <= set(p.thread_meshcop.tlv.type) and\
+                   p.mle.tlv.active_tstamp == 100 and\
+                   p.thread_meshcop.tlv.chan_mask_mask == '001fffc0' and\
+                   p.thread_meshcop.tlv.xpan_id == '000db80000000001' and\
+                   p.thread_meshcop.tlv.net_name == 'TEST_1' and\
+                   p.thread_meshcop.tlv.pskc == 'd2aa9cd8dff7919122d77d37ec3c1b5f'
```
Comment (simonlingoogle) [inline tests/scripts/thread-cert/Cert_9_2_05_ActiveDataset.py:249]: define constant in this script?

**Review by simonlingoogle — APPROVED** [review]
LGTM :+1: 

with nits

File: tests/scripts/thread-cert/Cert_9_2_05_ActiveDataset.py (hunk 1 above)
[the remark this answers is not in the document]
Comment (FiveDimensions) [reply tests/scripts/thread-cert/Cert_9_2_05_ActiveDataset.py:105]: there is no pskc and security_policy args in send_mgmt_active_set, so I put them into the binary

File: tests/scripts/thread-cert/Cert_9_2_05_ActiveDataset.py (hunk 2 above)
Comment (FiveDimensions) [reply tests/scripts/thread-cert/Cert_9_2_05_ActiveDataset.py:253]: add more comments

**Review by jwhui — APPROVED** [review]

**Pull request merged**